src/Microsoft.TestPlatform.Build/Microsoft.TestPlatform.targets

```diff
@@ -44,6 +44,7 @@ Copyright (c) .NET Foundation. All rights reserved.
       VSTestConsolePath="$(VSTestConsolePath)"
       VSTestResultsDirectory="$(VSTestResultsDirectory)"
       VSTestVerbosity="$(VSTestVerbosity)"
+      VSTestCollect="$(VSTestCollect)"
```

Comment:
Suggestion for code readability : We can arrange these properties in sorted order.